lib/foreman_tasks/engine.rb

```diff
@@ -113,6 +113,7 @@ class Engine < ::Rails::Engine
         world.middleware.use Actions::Middleware::KeepCurrentUser, :before => ::Dynflow::Middleware::Common::Transaction
         world.middleware.use Actions::Middleware::KeepCurrentTimezone
         world.middleware.use Actions::Middleware::KeepCurrentRequestID
+        world.middleware.use ::Actions::Middleware::LoadSettingValues
```

Comment:
Could we register the middleware only on 2.5 and newer instead of checking the version all the time?

Reply:
Well, I feel it's bit cleaner checking it in the middleware, as

1. it will look ugly in code
2. it is less prone to version not being initialized yet in this phase

OTH doing it in initializer would be easier to spot and remove once we bump foreman requirement version

Reply:
Done